app/Http/Controllers/HomeController.php

```diff
@@ -8,9 +8,16 @@ class HomeController extends Controller
 {
     public function index()
     {
-        $movies = Movie::all()->sortByDesc(function($movie) {
-            return $movie->ratings->avg('rating');
-        })->take(100);
+
+
+        $movies = Movie::query() //Using query for the next functions to be indented
```

Comment:
I really really like that, I've been using `query()` as well, I'm glad I'm not alone @Thotsuya :)